### ISSUE TITLE
Output different since 2.1.0?

### ISSUE THREAD

**Issue by wltng** [TheFes/cheapest-energy-hours#45]
I use 2 of your templates to tell me how long before cheapest energy hours start (which works great btw!).
Like so:

```
{% from 'cheapest_energy_hours.jinja' import cheapest_energy_hours %}
{% from 'relative_time_plus.jinja' import relative_time_plus %}
{% set time = cheapest_energy_hours('sensor.nordpool_kwh_nl_eur_3_09_0', hours=2, include_tomorrow=true, look_ahead=true) %}
{{ relative_time_plus(time, parts=2, week=false, time=true, abbr=true) }}
```

However, since cheapest-energy-hours was updated to version 2.1.0, I get an error "Invalid date" from relative-time-plus

If I copy the output of CEH and past it (as a string) in the RTP function, it does work...

```
{% from 'cheapest_energy_hours.jinja' import cheapest_energy_hours %}
{% from 'relative_time_plus.jinja' import relative_time_plus %}
{% set time = cheapest_energy_hours('sensor.nordpool_kwh_nl_eur_3_09_0', hours=2, include_tomorrow=true, look_ahead=true) %}
{{ time }}
{{ relative_time_plus( time ) }}
{{ relative_time_plus( '2023-10-05T03:00:00+02:00' ) }}
```

gives:
```
2023-10-05T03:00:00+02:00
Invalid date
6 hours
```
Am I using it wrong? Converting to timestamp doesn't seem to work either.

**Comment by TheFes** [comment]
Sorry about that, I didn't properly check if all my open and close tags were correct.
This is corrected in v3.0.0 which I just published.

**Comment by wltng** [comment]
Just updated to 3.0.0 and it works as expected again.
Thanks!
